include/snowflake/platform.h

```diff
@@ -41,7 +41,6 @@ typedef pthread_rwlock_t SF_RWLOCK_HANDLE;
 typedef pthread_mutex_t SF_MUTEX_HANDLE;
 
 #define PATH_SEP '/'
-//MAX PATH in linux is 4096
 #define MAX_PATH 4096
```

Comment:
For Windows platform its already defined in windows headers

Reply:
There is actually a variable defined for this in Linux called `PATH_MAX` in limits.h. I think what you want to do here is:

`#define MAX_PATH PATH_MAX`